config/post_update/30002/oe_corporate_blocks.ec_data.footer.yml

```diff
@@ -0,0 +1,80 @@
+langcode: en
```

Comment:
I've seen on oe_theme that also a short desc is added to the folder names, this would be something like 30002_update_footer_data. Dono if its mandatory but..